src/accel/SharedParams.cc

```diff
@@ -771,38 +763,41 @@ void SharedParams::set_num_streams(unsigned int num_streams)
 /*!
  * Write available Celeritas output.
  *
- * This can be done multiple times, overwriting the same file so that we can
- * get output before construction *and* after.
+ * The first call should provide an output filename. If it's to a file, an
+ * initial copy of the output (the setup) will be written in case of later
+ * failure. If the file already exists, a different filename will be used.
```

Comment:
Is it possible to make the unique filename an option that's disabled by default? It can create a lot of file clutter, and it seems easy enough to change the output filename when needed in a script.

Reply:
The motivating reason for this PR is that it's super easy to run a bunch of Geant4 jobs with the macros and unwittingly lose every output but the last, so I'm hesitant to make the default to overwrite. Maybe I didn't think this through, but perhaps a middle ground (easier to implement too if I'd thought of it before) would be just to generate a default timestamped filename if the user doesn't specify a filename in the input? Should we still make uniquifying an option?

Reply:
I suppose, although any other output (e.g. root output) would still be overwritten, and it also seems super easy for the user to adjust the output filenames to be unique :). I do like that idea for a unique default filename better since if a user does specify a filename the output will be what's expected.

Reply:
Good points! I'll make those changes.